src/edu/pcc/cis/helloworld/HelloWorld.java

```diff
@@ -1,4 +1,14 @@
 package edu.pcc.cis.helloworld;
+import java.util.Date;
+
+/**
+ * @author: Sam Gonzales
+ * @version: 2018.05.01
+ */
 
 public class HelloWorld {
+    public static void main(String[] args) {
+        System.out.println("Hello, World");
```

Comment:
Nice job. Making a comment so you can see how it appears inline in a pull request.